src/plugins/explore/public/components/patterns_table/constants.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HorizontalAlignment } from '@elastic/charts';
+import { PatternItem } from './patterns_table';
+
+export const PATTERNS_FIELD = 'patterns_field';
+export const COUNT_FIELD = 'count';
+
+export const patternsTableColumns = [
+  {
+    field: 'ratio',
+    name: 'Event ratio',
+    render: (val: number) => {
+      // Check if value is a valid, finite number before formatting
+      if (typeof val !== 'number' || isNaN(val) || !isFinite(val)) {
+        return '-';
+      }
+      return `${(val * 100).toFixed(2)}%`;
+    },
+    width: '15%',
+  },
+  {
+    field: 'pattern',
+    name: 'Pattern',
+    render: (pattern: string) => pattern || '-',
+  },
+  {
+    field: 'count',
+    name: 'Event count',
```

Comment:
would these need traslation?

Reply:
yeah you can use i18n. search within explore for `i18n` and you'll see several ways to do it.

all texts that user sees should be translated